jans-lock/cedarling/README.md

```diff
@@ -7,10 +8,81 @@ For example, why display form fields that a user is not authorized to see?
 The Cedarling is a more productive and flexible way to handle authorization.
 
 ## Rust Cedarling
+
 Cedarling is written in the Rust programming language (folder `cedarling`). And you can import it into your project as a dependency.
 
 ## Cedarling bindings
+
 We have support binding for this platforms:
+
 - [ ] Python
 - [ ] Wasm
 
+## Examples of rust Cedarling
+
+Rust examples of using Cedarling contains in the folder `cedarling/examples`.
+
+### Example of initialization logger
+
+File with example is `log_init.rs`.
+We support 4 types of loggers according to the [bootstrap properties](https://github.com/JanssenProject/jans/wiki/Cedarling-Nativity-Plan#bootstrap-properties).
+
+To run example you need execute next command according to the logger:
+
+- `off` - This log type is do nothing. It means that all logs will be ignored.
+
+```bash
+cargo run -p cedarling --example log_init -- off
+```
+
+- `stdout`- This log type writes all logs to `stdout`. Without storing or additional handling log messages.
+[Standart streams](https://www.gnu.org/software/libc/manual/html_node/Standard-Streams.html).
+
+```bash
+cargo run -p cedarling --example log_init -- stdout
+```
+
+- `memory` - This log type holds all logs in database (in memory) with eviction policy.
+
+```bash
+cargo run -p cedarling --example log_init -- memory 60
+```
+
+60 - ttl (time to live log entry) in seconds.
+But actually the example execute very fast, so we no need to wait.
+
+- `lock` - This log type will send logs to the server (corporate feature). (in development)
+
+```bash
+cargo run -p cedarling --example log_init -- lock
+```
+
+## Code coverage
```

Comment:
can you also include the commands to build, run and test Cedarling.

Reply:
Added information how to run unit tests. 
And we have nothing to build because we have library and have no bindings to python for now.

How to run examples, it was explained.